src/main/java/taxi/dao/jdbc/CarDaoJdbc.java

```diff
@@ -123,8 +125,12 @@ private Driver createDriverInstance(ResultSet resultSet) throws SQLException {
         Long driverId = resultSet.getObject("d_id", Long.class);
         String name = resultSet.getObject("d_name", String.class);
         String licenseNumber = resultSet.getObject("license_number", String.class);
+        String login = resultSet.getObject("login", String.class);
+        String password = resultSet.getObject("password", String.class);
```

Comment:
let's not show users' password when we fetch data from db

Reply:
It is used only for construction of Driver instance, it is not used in toString() or in .jsp files

Reply:
In the future, you will use DTOs but in generally there is no point of passing password from backend to frontEnd
(for now it's optional to change it)